Angular/src/app/navbar/navbar.component.html

```diff
@@ -1,11 +1,9 @@
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
  integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
-<section class="container">
-    <img src="../../assets/Kainos_Logo.png" style="height: 80px;" alt="Kainos Logo">
```

Comment:
Avoid inline styling, use an ID and do the styling in css if its for something specific